### PR TITLE
Post single product 8

### PR DESCRIPTION
**Pull request by ivanfex**
### Assignee Tasks

- [ ] added unit tests (or none needed)
- [ ] written relevant docs (or none needed)
- [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

User is able to post a new product from ~/products/add

### CODE REVIEW

File: client/components/AddProduct.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        })
+    }
+
+    handleSubmit(event){
+        event.preventDefault()
+        this.props.addProduct(this.state)
+    }
+
+    render(){
+        console.log(this.state)
```
Comment (Thanh-Lai) [inline client/components/AddProduct.js]: Take away the console.log

File: client/components/AddProduct.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        return (
+            <div>
```
Comment (Thanh-Lai) [inline client/components/AddProduct.js]: I think we should add a Header like <h1>Add a new Product</h1>

File: client/components/AddProduct.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                    <label>Image URL</label>
+                    <input name="imageUrl" onChange={this.handleChange}/>
+                    <button type="submit">Submit</button>
+                </form>
+            </div>
+        )
+    }
+}
+
+const mapState = ()=>({})
```
Comment (Thanh-Lai) [inline client/components/AddProduct.js:58]: Do we need a mapState if we dont use it?

Comment (ivanfex) [reply client/components/AddProduct.js:58]: We still need to pass two arguments to the connect function

File: client/components/AddProduct.js (hunk 2 above)
Comment (ivanfex) [reply client/components/AddProduct.js]: Done

Comment (ivanfex) [reply client/components/AddProduct.js]: This is just to test functionality the form will in a modal later

File: client/components/AddProduct.js (hunk 1 above)
Comment (ivanfex) [reply client/components/AddProduct.js]: Done

**Review by jessdelacruzsantos — APPROVED** [review]
Super awesome, works like a dream.

**Review by Sampandu — APPROVED** [review]

**Pull request merged**